statesman-server/docker-compose.yml

```diff
@@ -29,6 +30,7 @@ services:
       DB_PASSWORD: password
       FOXTROT_ENDPOINT: https://localhost:17000
       FOXTROT_TOKEN: INV_TOKEN_
+      # JAVA_DEBUG_OPTS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8090
```

Comment:
The compose is only used for local dev. This can be enabled always. Uncomment this

Reply:
Done